SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -87,6 +98,7 @@ Git {
 			cmd = ["git"];
 		});
 		cmd = (cmd ++ args).join(" ");
+		// this blocks the app thread
```

Comment:
we need another way to get the output of a command. Probably I can change it to use Pipe but that still blocks while its in getLine.